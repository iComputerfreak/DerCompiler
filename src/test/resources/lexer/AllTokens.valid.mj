abstract
assert
boolean
break
byte
case
catch
char
class
const
continue
default
double
do
else
enum
extends
false
finally
final
float
for
goto
if
implements
import
instanceof
interface
int
long
native
new
null
package
private
protected
public
return
short
static
strictfp
super
switch
synchronized
this
throws
throw
transient
true
try
void
volatile
while
!=
!
(
)
*=
*
++
+=
+
,
-=
--
-
.
/=
/
:
;
<<=
<<
<=
<
==
=
>=
>>=
>>>=
>>>
>>
>
?
%=
%
&=
&&
&
[
]
^=
^
{
}
~
|=
||
|
abcdef
_abccdef_
0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
30
40
50
100
1000
10000
100000
1000000
10000000
100000000
2147483647
-2147483648
