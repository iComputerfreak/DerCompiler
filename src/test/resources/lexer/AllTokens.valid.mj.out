abstract
assert
boolean
break
byte
case
catch
char
class
const
continue
default
double
do
else
enum
extends
false
finally
final
float
for
goto
if
implements
import
instanceof
interface
int
long
native
new
null
package
private
protected
public
return
short
static
strictfp
super
switch
synchronized
this
throws
throw
transient
true
try
void
volatile
while
!=
!
(
)
*=
*
++
+=
+
,
-=
--
-
.
/=
/
:
;
<<=
<<
<=
<
==
=
>=
>>=
>>>=
>>>
>>
>
?
%=
%
&=
&&
&
[
]
^=
^
{
}
~
|=
||
|
identifier abcdef
identifier _abccdef_
integer literal 0
integer literal 1
integer literal 2
integer literal 3
integer literal 4
integer literal 5
integer literal 6
integer literal 7
integer literal 8
integer literal 9
integer literal 10
integer literal 11
integer literal 12
integer literal 13
integer literal 14
integer literal 15
integer literal 16
integer literal 17
integer literal 18
integer literal 19
integer literal 20
integer literal 30
integer literal 40
integer literal 50
integer literal 100
integer literal 1000
integer literal 10000
integer literal 100000
integer literal 1000000
integer literal 10000000
integer literal 100000000
integer literal 2147483647
-
integer literal 2147483648
EOF
